app/models/host/base.rb

```diff
@@ -181,7 +181,7 @@ def set_interfaces(parser)
 
       ipmi = parser.ipmi_interface
       if ipmi.present?
-        existing = self.interfaces.where(:mac => ipmi[:macaddress], :type => Nic::BMC).first
+        existing = self.interfaces.where(:mac => ipmi[:macaddress], :type => Nic::BMC.name).first
```

Comment:
not that it makes much difference, but why not just `"Nic::BMC"`?

Reply:
I guess it was done for the sake of diff-minimizing, but IMHO using the Constant instead of a string is a tad nicer.

Reply:
most other places currently pass the sting, but WFM, not worth the effort..